python/pyfunc-server/echo-model/__init__.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import grpc
+import mlflow
+from caraml.upi.v1 import upi_pb2
+from merlin.model import PyFuncModel
+
+
+class EchoUPIModel(PyFuncModel):
+    def __init__(self, model_name, model_version):
+        self._model_name = model_name
+        self._model_version = model_version
+
+    def infer(self, request):
+        return request
+
+    def upiv1_infer(self, request: upi_pb2.PredictValuesRequest,
```

Comment:
I don't see this function being declared (as empty) in PyFuncModel. How should we enforce that user is declaring the function with a correct name and params? Are we solely relying on documentations?

Reply:
What do you mean? It's in here https://github.com/gojek/merlin/blob/4c4d473277f9777ad7014fe80eb6a5433daab96d/python/sdk/merlin/pyfunc.py#L99